flink-state-backends/flink-statebackend-changelog/src/test/resources/log4j2-test.properties

```diff
@@ -18,11 +18,11 @@
 
 # Set root logger level to OFF to not flood build logs
 # set manually to INFO for debugging purposes
-rootLogger.level = OFF
+rootLogger.level = INFO
```

Comment:
The common approach is to disable logging by default and change level on demand. Can't we use it here?

ditto: pattern

Reply:
BTW, what's the problem for `layout.pattern`? Do you mean we should remove it?

Reply:
> I will change the logging level back.

Thanks!
> BTW, what's the problem for layout.pattern? Do you mean we should remove it?

Actually, the change is correct, non-standard pattern was used before, sorry. Let's keep this change.